docs/manual/config/lazy.rst

```diff
@@ -73,7 +73,9 @@ Group functions
     * - ``lazy.screen.toggle_group()``
       - Move to the last visited group
     * - ``lazy.group["group_name"].toscreen()``
-      - Move to the group called ``group_name``
+      - Move to the group called ``group_name``.
+        Takes an optional ``toggle`` parameter (defaults to True).
+        If this group is already on the screen, then the group is toggled
```

Comment:
Maybe better:
If this group is already on the screen, then the group is toggled with last used

Reply:
I just copied it straight from the source code comments.  But I am up for whatever.